packages/mailer/src/GenericEmail.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Tempest\Mail;
+
+/**
+ * Represents a generic email.
+ */
+final class GenericEmail implements Email
```

Comment:
If the plan is still to encourage people to make their own Email implementations (I think they should), then we should provide a trait `IsEmail` as well for convenience. Certainly if/when `Envelope` and `Content` get refactored out.

Reply:
Maybe we shouldn't even have that `GenericEmail`, and encourage either creating email classes or using the email builder

In both cases, an `IsEmail` trait might indeed come in handy

Reply:
> Maybe we shouldn't even have that GenericEmail, and encourage either creating email classes or using the email builder

I like that :) Let's do that!